src/Mapbender/ManagerBundle/Resources/views/Application/form-elements.html.twig

```diff
@@ -88,7 +88,7 @@
                     <i class="fas fa-key"></i>
                 </span>
                 <span class="-fn-delete hover-highlight-effect"
-                      title="{{ 'mb.actions.delete' | trans }}"
+                      title="{{"mb.manager.admin.element.delete.title"|trans}} {{ element_title(element) | trans }}"
```

Comment:
As Astrid already said: Please add a template parameter here as well

Reply:
The translations of the text for deleting the user were no problem. But the text for deleting an element could not be translated easily. The latter only worked after the translation text was moved to another yaml-file. Furthermore, I could not find any other examples in the source code in which texts from e.g. application/mapbender/src/Mapbender/ManagerBundle/Resources/translations/messages+intl-icu.de.yaml, which are composed of several parts in a twig-template, are correctly translated or applied. 
Do you have any idea why this is the case?